datahub-web-react/src/app/sharedV2/OverflowList.tsx

```diff
@@ -17,9 +17,7 @@ const HiddenContainer = styled.div`
     pointer-events: none;
 `;
 
-const FitContainer = styled.div`
-    width: fit-content;
-`;
+const FitContainer = styled.div``;
```

Comment:
sadly I'm still seeing this issue after pulling your branch down :( idk exactly what's going on here..

since this is a minor edge case, let's just timebox this and if we can't figure it out move the ticket back into to do. however since we have the time and context let's give it a little more time right now trying to figure out what's wrong